microcosm_flask/fields/query_string_list.py

```diff
@@ -18,17 +18,19 @@ def _deserialize(self, value, attr, obj, **kwargs):
 
         /foo?bars=1,2
         /foo?bars[]=1&bars[]2
+        /foo?bars=&...
 
         and returns a list of values
 
         """
         if value is None:
             return None
+        if value == "":
+            return []
```

Comment:
This is the main change.

Reply:
This will break the `value == str(deserialize(value))` symmetry. Not sure if this is a problem.

Reply:
In the case of a query string, I don't think there's expectation of such a symmetry. One example is that a query string list can be passed as `foo[]=a&foo[]=b`